### PR TITLE
Qualifying DC/OS 1.13.3 against RHEL 7.5 and Docker 18.09.2

### PR DESCRIPTION
**Pull request by VerginiyaStoyanova**
JIRA: https://jira.mesosphere.com/browse/DCOS-56228
Results: 209 passed, 8 skipped, 2 warnings in 4142.21 seconds

### CODE REVIEW

File: rhel/7.5/aws/DCOS-1.13.3/docker-18.09.2-ce/selinux_disabled/packer.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+    "region":         "us-west-2"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "instance_type": "m4.xlarge",
+      "source_ami": "ami-036affea69a1101c9",
```
Comment (ahairape) [inline rhel/7.5/aws/DCOS-1.13.3/docker-18.09.2-ce/selinux_disabled/packer.json]: This seems to be a wrong AMI ID. The one for RHEL 7.5 is ami-28e07e50

File: rhel/7.5/aws/DCOS-1.13.3/base_images.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "us-west-2": "ami-ami-28e07e50"
```
Comment (ahairape) [inline rhel/7.5/aws/DCOS-1.13.3/base_images.json]: This AMI ID should be "ami-28e07e50"

File: rhel/7.5/aws/DCOS-1.13.3/docker-18.09.2-ce/selinux_disabled/packer.json (hunk 1 above)
Comment (VerginiyaStoyanova) [reply rhel/7.5/aws/DCOS-1.13.3/docker-18.09.2-ce/selinux_disabled/packer.json]: Changed!

File: rhel/7.5/aws/DCOS-1.13.3/base_images.json (hunk 2 above)
Comment (VerginiyaStoyanova) [reply rhel/7.5/aws/DCOS-1.13.3/base_images.json]: Changed!

**Pull request merged**